packages/playwright-core/src/server/bidi/bidiBrowser.ts

```diff
@@ -199,6 +199,12 @@ export class BidiBrowserContext extends BrowserContext {
     promises.push(this.doUpdateDefaultViewport());
     if (this._options.geolocation)
       promises.push(this.setGeolocation(this._options.geolocation));
+    if (this._options.locale) {
+      promises.push(this._browser._browserSession.send('emulation.setLocaleOverride', {
+        locale: this._options.locale,
+        userContexts: [this._browserContextId || 'default'],
```

Comment:
```suggestion
        userContexts: [this._userContextId()],
```
Maybe you can also replace other usages of `this._browserContextId || 'default'` in the file while you are here, to avoid confusion in the future?

Reply:
Done